main.js

```diff
@@ -0,0 +1,132 @@
+const dinos = [{
+  name: "Steve Stegosaurus",
+  id: 1,
+  dob: "01/02/0000",
+  bio: "Steve is super rad and loves long walks on the beach.",
+  imageUrl: "https://cdn.shopify.com/s/files/1/2281/5369/products/30002_1_x700.jpg?v=1519820343",
+  isAvailable: true,
+  isCarnivore: false,
+  type: "Stegosaurus"
+},
+{
+  name: "Brian Brachiosaurus",
+  id: 2,
+  dob: "01/03/0000",
+  bio: "I'm super tall (over 9 inches!) and I have a huge brain.",
+  imageUrl: "https://i.ebayimg.com/images/g/hWAAAOSwd-panuG0/s-l1600.jpg",
+  isAvailable: true,
+  isCarnivore: false,
+  type: "Brachiosaurus"
+},
+{
+  name: "Ted Triceratops",
+  id: 3,
+  dob: "01/04/0000",
+  bio: "Just looking for love!",
+  imageUrl: "https://ssli.ebayimg.com/images/g/0NMAAOSwpcBao7ep/s-l640.jpg",
+  isAvailable: false,
+  isCarnivore: false,
+  type: "Triceratops"
+},
+{
+  name: "Maurice Mammoth",
+  id: 4,
+  dob: "01/05/0000",
+  bio: "Fake Dinosaur",
+  imageUrl: "https://i.ebayimg.com/images/g/Up8AAOSwICxaoW-k/s-l1600.jpg",
+  isAvailable: true,
+  isCarnivore: false,
+  type: "Impostor"
+},
+{
+  name: "Richard Rex",
+  id: 5,
+  dob: "01/06/0000",
+  bio: "Richard enjoys long walks in the sun and drinking Pina Coladas.",
+  imageUrl: "https://i.ebayimg.com/images/g/RhIAAOSwFb5aG7wo/s-l1600.jpg",
+  isAvailable: true,
+  isCarnivore: true,
+  type: "T-Rex"
+},
+{
+  name: "Sally Spinosaurus",
+  id: 6,
+  dob: "01/07/0000",
+  bio: "Former contestant on the bachlorette.",
+  imageUrl: "https://i.ebayimg.com/images/g/pu0AAOSwGjZankAW/s-l1600.jpg",
+  isAvailable: false,
+  isCarnivore: true,
+  type: "Spinosaurus"
+},
+{
+  name: "Danielle Dimetrodon",
+  id: 7,
+  dob: "01/08/0000",
+  bio: "",
+  imageUrl: "https://ssli.ebayimg.com/images/g/RGgAAOSwWGxao8Ra/s-l640.jpg",
+  isAvailable: false,
+  isCarnivore: true,
+  type: "Dimetrodon"
+},
+{
+  name: "Viserion",
+  id: 8,
+  dob: "01/09/0000",
+  bio: "Viserion is a dragon that the Night King slayed and reanimated as a wight. He is one of the three dragons born in the Dothraki Sea, along with Drogon and Rhaegal, and is named after Daenerys Targaryen's elder brother, Viserys. Before becoming a wight, he can be distinguished by his cream and gold colored scales, and red-orange wings.",
+  imageUrl: "https://i.ebayimg.com/images/g/jlYAAOSwpCxZ5T9b/s-l1600.png",
+  isAvailable: true,
+  isCarnivore: false,
+  type: ""
+},
+{
+  name: "Terry Therizinosaurus",
+  id: 9,
+  dob: "01/10/0000",
+  bio: "I am full of feelz.",
+  imageUrl: "https://ssli.ebayimg.com/images/g/mVAAAOSwDdNao7G3/s-l640.jpg",
+  isAvailable: true,
+  isCarnivore: false,
+  type: "Therizinosaurus"
+},
+{
+  name: "Al Acrocanthosaurus",
+  id: 10,
+  dob: "01/11/0000",
+  bio: "NOPE NOPE NOPE",
+  imageUrl: "https://i.ebayimg.com/images/g/l3EAAOSwMQhaet~m/s-l1600.jpg",
+  isAvailable: false,
+  isCarnivore: true,
+  type: "Acrocanthosaurus"
+}];
+
```

Comment:
@zoeames - stop putting in all these extra spaces. jeez.

Reply:
oops fixed it